lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -1968,152 +1983,320 @@ Error TFLiteModelLoader::loadSlice(const tflite::Operator *op,
   return setOutputNodeValue(op, output);
 }
 
-Error TFLiteModelLoader::loadTile(const tflite::Operator *op,
-                                  const OperatorInfo &opInfo) {
+Error TFLiteModelLoader::loadSpaceToBatchNd(const tflite::Operator *op,
+                                            const OperatorInfo &opInfo) {
   NodeValue input;
   ASSIGN_VALUE_OR_RETURN_ERR(input, getInputNodeValue(op, 0));
-  NodeValue multiples;
-  ASSIGN_VALUE_OR_RETURN_ERR(multiples, getInputNodeValue(op, 1));
+  NodeValue block;
+  ASSIGN_VALUE_OR_RETURN_ERR(block, getInputNodeValue(op, 1));
+  NodeValue pads;
+  ASSIGN_VALUE_OR_RETURN_ERR(pads, getInputNodeValue(op, 2));
+  TypeRef outTy;
+  ASSIGN_VALUE_OR_RETURN_ERR(outTy, getOutputType(op, 0));
 
-  auto numDims = input.getType()->dims().size();
-  std::vector<unsigned_t> numTiles;
-  ASSIGN_VALUE_OR_RETURN_ERR(numTiles,
-                             loadArray<unsigned_t>(opInfo, multiples));
-  RETURN_ERR_IF_NOT(numTiles.size() == numDims,
-                    opErrMsg(opInfo, "Input operand 'multiples' length should "
-                                     "match the number of input dimensions!"));
+  // Implemented as Pad -> Transpose N-D -> SpaceToDepth -> Transpose N-D
```

Comment:
Could you create a Graph.cpp utility for creating this sequence of nodes e.g. ```Node *createSpaceToBatchNd(...))```.
This makes the logic reusable but also this will allow us later (if necessary) to add a ```SpaceToBatchNode``` and a lowering logic for it.

Reply:
Makes sense, but I suggest this is left for another PR. I'd rather skip working on this at the moment as this would have a very low priority for us, given that ONNX doesn't have these ops.